node/cn/tracers/api.go

```diff
@@ -98,13 +98,13 @@ type API struct {
 
 // NewAPI creates a new API definition for the tracing methods of the CN service,
```

Comment:
In the comment, `NewAPI` -> `NewAPIUnsafeDisabled`

Reply:
Resolved cf88fea